### PR TITLE
added AdGuard Home scripts

### PR DESCRIPTION
**Pull request by NathanAdhitya**
I use AdGuard Home and I've wanted to point domains in this repo to my LANCache.
So, I took the script from create-dnsmasq.sh and modified it.

Adguard Home Upstream File Reference: https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams-from-file

### CODE REVIEW

File: scripts/config.example.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
 	"ips": {
+		"adguardhome_upstream": ["tls://dns.google","https://dns.google/dns-query"],
```
Comment (VibroAxe) [inline scripts/config.example.json:3]: I'm not familar with adguardHome, does it support non DoH servers, if so, would be good to have an example of that here?

**Comment by VibroAxe** [comment]
See https://github.com/uklans/cache-domains/pull/194/files#r947641931

File: scripts/config.example.json (hunk 1 above)
Comment (NathanAdhitya) [reply scripts/config.example.json:3]: Hello, that is already mentioned on the same page, it supports much more than just DoH servers: https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams

But yeah, I like your idea.

**Comment by VibroAxe** [comment]
Looks good to me, thanks for the PR

**Pull request merged**